test/return.js

```diff
@@ -59,3 +59,21 @@ builder.add(function testResponseViaPromise(test) {
       test.equal(result.returns, response, 'Response should be returned through promise')
     })
 })
+
+// Make sure any A+-compliant promises are handled correctly.
+builder.add(function testAplusCompliant(test) {
+  this.graph.add('returns', function () {
+    return {
+      then: function (callback) {
```

Comment:
glad you asked! most promise libraries add utility methods. I wanted to make sure we didn't call any of the new methods accidentally. this is a minimally viable promise, per the spec